File: Data Structures & Algorithms/subsets/submission-1.java
class Solution {
    public List<List<Integer>> subsets(int[] nums) {
        List<List<Integer>> res = new ArrayList<>();
        List<Integer> sublist = new ArrayList<>();

        backtrack(nums, 0, res, sublist);

        return res;
    }

    public void backtrack(int[] nums, int start, List<List<Integer>> res, List<Integer> sublist) {
        res.add(new ArrayList<>(sublist));

        for ( int i = start; i < nums.length; i++) {
            sublist.add(nums[i]);
            backtrack(nums, i + 1, res, sublist);
            sublist.remove(sublist.size() - 1);
        }
    }
}

File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        dict_cnt=dict()
        for i in nums:
            dict_cnt[i]=dict_cnt.get(i,0)+1
        for k,v in dict_cnt.items():
            if(v>=int(len(nums)/2)):
                return k
        